### PR TITLE
Updating reward network harness with the version that trains

### PR DESCRIPTION
**Pull request by soumide1102**
Started from policy network and made incremental changes to modify training script to get to the reward network. This is the version that trains and includes hyperparameters in the csv file that produces a stable loss curve.

### CODE REVIEW

**Comment by soumide1102** [comment]
@hickmank Yes, this is ready for review and merge. Just merged upstream/main.
